settings.gradle

```diff
@@ -19,3 +19,5 @@ include "wingtips-core",
         "samples:sample-jersey2",
         "samples:sample-spring-web-mvc",
         "samples:sample-spring-boot"
+include 'wingtips-lightstep'
```

Comment:
Put this up right after `"wingtips-jersey2"` (before the test-only and sample modules).

Reply:
(and get rid of the `include` - just make it part of the existing include list)